wagtail/admin/static_src/wagtailadmin/js/core.js

```diff
@@ -100,7 +100,7 @@ $(function() {
         $('.nav-wrapper').css('min-height', $(window).height());
         $('.nav-main').each(function() {
             var thisHeight = $(this).height();
```

Comment:
Although it does not fit the scope of the pull-request, I noticed the by the change in the next-line the`thisHeight` variable is not being used. Is this something you could cleanup in this pull-request as well? :smile: